test/unit/sdam/server_selection.test.js

```diff
@@ -53,19 +115,19 @@ describe('server selection', function () {
             MIN_SECONDARY_WRITE_WIRE_VERSION,
             ReadPreference.secondary
           );
-          const server = selector(topologyDescription, Array.from(serverDescriptions.values()));
+          const servers = selector(topologyDescription, Array.from(serverDescriptions.values()));
```

Comment:
Server selectors return an array of server descriptions so I fixed the variable name in these tests to be plural.